packages/playground/assets/__tests__/assets.spec.ts

```diff
@@ -162,7 +162,9 @@ describe('svg fragments', () => {
 
   test('from js import', async () => {
     const img = await page.$('.svg-frag-import')
-    expect(await img.getAttribute('src')).toMatch(/svg#icon-heart-view$/)
+    expect(await img.getAttribute('src')).toMatch(
+      isBuild ? /svg%3e#icon-heart-view$/ : /svg#icon-heart-view$/
```

Comment:
Where does the `%3e` come from, and why only in build mode?

Reply:
@aleclarson it's the url-encoded version of `>` so the string being matched is `svg>#icon-heart-view`, whereas the non-build mode version hasn't had the SVG inlined, thus is still referencing the SVG by filename, instead of the closing `svg` tag, it's the `svg` extension.

I'm assuming this was tested in a browser that it actually works with the fragment and not just to pass the test though. I know that `#` needs to be url-encoded for inlining SVG usually, but perhaps it's not meant to be when appending the fragment identifier after the inlined SVG like you would append it after a file extension? I've never used fragments, but I'm interested to know if that works as intended.